connaisseur/tests/integration/values.yaml

```diff
@@ -0,0 +1,72 @@
+# configure connaisser deployment
```

Comment:
Do we need the duplicate here or could we alter the "real" `values.yaml` file directly to prevent that changes to the `values.yaml` are forgotten to be updated here?

Reply:
Thought it would be easier this way in the sense that moving this prepared values is a single `envsubst` step, while altering the `values.yaml` will be multiple steps (which may become more over time) that could fail but have nothing to do with the deployment.
In both cases though, one would immediately get the feedback on the PR's first pipefile, so neither should be a problem.

So choice was deliberate, but I'm not strongly opinionated. If, given the above consideration,you still feel altering > duplication, I'll change it

Reply:
Hmm, you're right, that would increase the lines of code in the ci/cd file :thinking: 
Due to some bad experience with duplicated code I would still tend to altering, but also get your point.